src/core/globals.ml

```diff
@@ -66,6 +69,11 @@ let platform_name = function
 	| Python -> "python"
 	| Hl -> "hl"
 	| Eval -> "eval"
+	| CustomTarget c -> "custom_" ^ c
+
+let platform_define = function
```

Comment:
Why can't we just make `platform_name` output the target name without the `custom_` prefix? That's what I'd expect it to do, and then we don't need this extra function.